onnxruntime/core/platform/telemetry.cc

```diff
@@ -12,6 +12,20 @@ void LogRuntimeError(uint32_t sessionId, const common::Status& status, const cha
   env.GetTelemetryProvider().LogRuntimeError(sessionId, status, file, function, line);
 }
 
+bool Telemetry::IsEnabled() const {
+  return false;
+}
+
+// Get the current logging level
+unsigned char Telemetry::Level() const {
```

Comment:
Is there a reason why this is unsigned char vs something more expected like uint8_t?

Reply:
The Level defined has uchar is coming from the [ETW Enable callback](https://learn.microsoft.com/en-us/windows/win32/api/evntprov/nc-evntprov-penablecallback) in [TraceLoggingRegisterEx](https://learn.microsoft.com/en-us/windows/win32/api/traceloggingprovider/nf-traceloggingprovider-traceloggingregisterex). I agree it's not as intuitive as what you suggested and had a similar intuition but left defined as is

Reply:
Comment capturing this would be good.